build.gradle

```diff
@@ -38,15 +74,25 @@ android {
     lintOptions {
         disable 'InvalidPackage'
     }
+
 }
 
+afterEvaluate {
+  android.applicationVariants.all { variant ->
+    variant.javaCompiler.dependsOn(Clone)
+  }
+}
+
+Clone.finalizedBy(BuildCJDNS)
+BuildCJDNS.finalizedBy(CopyT)
```

Comment:
We run android_do on every build, so, sort of yes but no :P It will rebuild it only if you have changes. And it is sort of configurable because you can comment line.